addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -358,23 +304,7 @@ options.registry.BlogPostTagSelection = options.Class.extend({
      * @override
      */
     async _renderCustomXML(uiFragment) {
-        const $tagList = $(uiFragment.querySelector('.o_wblog_tag_list'));
-        for (const tagID of this.tagIDs) {
-            const tag = this.allTagsByID[tagID];
-            $tagList.append(qweb.render('website_blog.TagListItem', {
-                tag: tag,
-            }));
```

Comment:
Actually you are also removing the use of this template but you do not remove its definition. Please be careful about this, it is very hard to catch in reviews although very easy to remember to do during development :wink: 

It is also still the case for the template `wysiwyg.widgets.image.optimize` that was recently reported to me as dead code left behind during migration to right panel :wink: I still have to remove that :grimacing:

Reply:
EDIT: the optimize dead code is being removed as part of this PR https://github.com/odoo/odoo/pull/65828/commits/d30cf915eb9d6b3eecc04b010cba169ba6b3b00e :wink: